ManageCoursesUi.Tests/CourseControllerTests.cs

```diff
@@ -35,6 +35,46 @@ public class CourseControllerTests
                           model.OrganisationName == TestHelper.OrganisationName);
         }
         [Test]
+        [TestCase(EnumDataType.SingleVariantOneMatch)]
+        [TestCase(EnumDataType.MultiVariantOneMatch)]
+        public async Task TestController_Variants_with_lower_case_parameters_should_return_matched_model(EnumDataType type)
+        {
+            var manageApi = new Mock<IManageApi>();
+            var testData = TestHelper.GetTestData(type, null, null);
+
+            manageApi.Setup(x => x.GetCoursesByOrganisation(It.IsAny<string>())).ReturnsAsync(testData);
```

Comment:
This doesn't test anything - the system that was exhibiting the bug is being mocked out and returns the correct result set regardless of inputs. we shouldn't merge this PR